tests/ProxyManagerTest/Functional/RemoteObjectFunctionalTest.php

```diff
@@ -33,12 +36,15 @@
 final class RemoteObjectFunctionalTest extends TestCase
 {
     /**
-     * @param mixed   $expectedValue
+     * @param mixed $expectedValue
+     * @param string $method
      * @param mixed[] $params
+     * @return XmlRpcAdapter
      */
-    protected function getXmlRpcAdapter($expectedValue, string $method, array $params) : XmlRpcAdapter
+    protected function getXmlRpcAdapter($expectedValue, string $method, array $params): XmlRpcAdapter
```

Comment:
Given the context below, let's rename `$params` in `$parametersExpectedByClient`

Reply:
Done.